data/obs-packaging/clear-containers-image/README.md

```diff
@@ -0,0 +1,9 @@
+#clear-containers image
+
+This source rpm is the mini-os image used to boot clear-containers. 
+To update and generate an rpm spec and dsc to create rpm and deb packages, 
+you need to run:
+
+``./update_image.sh``
+
+By default it will generate the specs using the version specified in ../../../versions.txt
```

Comment:
Maybe add backticks for `version.txt`. It might be simpler to say that the file `versions.txt` at in the root of the repository rather than all the `../../` bits.